ldms/src/store/store_app/store_app.c

```diff
@@ -332,6 +334,8 @@ __get_sos_schema(store_app_cont_t cont, const char *name,
 	if (!schema) {
 		/* Create a new schema for 'name' */
 		struct sos_schema_template *tmp = &app_schema_template;
+		uuid_generate_random(uuid);
```

Comment:
If we're generating random uuid, then it's not really useful for ensuring that schema across containers and partitions are the same. If we have multiple aggregators storing data for the same schema, they will have different UUID which means that attempting to combine these partitions later into a single container for analysis will fail...

Reply:
You're right. This should be the LDMS schema digest, and this patch must be on top of the LDMS schema digest patch. I'll close this pull request put this change on top of the schema digest commit.